openedx/core/djangoapps/theming/tests/test_theme_style_overrides.py

```diff
@@ -64,6 +64,69 @@ def test_logo_image(self):
         result = staticfiles.finders.find('test-theme/images/logo.png')
         self.assertEqual(result, settings.TEST_THEME / 'lms/static/images/logo.png')
 
+    @with_comprehensive_theme("test-theme")
+    def test_override_block_in_parent(self):
+        """
+        Test that theme title is used instead of parent title.
+        """
+        self._login()
+        dashboard_url = reverse('dashboard')
+        resp = self.client.get(dashboard_url)
+        self.assertEqual(resp.status_code, 200)
+        # This string comes from the 'pagetitle' block of the overriding theme.
+        self.assertContains(resp, "Overridden Title!")
```

Comment:
I think you should also `self.assertContains(resp, "Explore courses")` or similar, to check that we're getting the full dashboard with our modifications on top, not just a blank template with the content of the overridden blocks.
Same in the other „dashboard“ test. Or this could be in its own test.

Reply:
I think the  test_parent_content_in_self_inherited_template test covers this case. They are testing the same theme with one test testing the overwritten fields and one testing the non overwritten fields.

Reply:
Yes, the `test_parent_content_in_self_inherited_template` covers this case, but this test did not yet exist when @clemente made that comment, I added it later.